LayoutTests/imported/w3c/web-platform-tests/html/interaction/focus/sequential-focus-navigation-and-the-tabindex-attribute/focus-tabindex-order.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE html><!-- webkit-test-runner [ TabsToLinks=true ] -->
```

Comment:
Please do not add those to WPT.

Reply:
It is needed to make it pass and also similar is used in `media` (track) related WPT tests.

Reply:
These are generally not exportable to WPT, and as a result, these annotations will generally be lost at the next import. These don't reflect on wpt.fyi either.

Generally, we deal with this in WPT by adding tabindex="0" to buttons, but  in this case, the test is intentionally testing a missing value or an empty value.

I'd rather have this test skipped, since it's not testing default shipping behavior.

Reply:
@nt1m - tackled it and now it is as we discussed. :-)